Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -1,58 +1,26 @@
 const SteamScript = include("engines.wine.quick_script.steam_script");
-
-const DotNET472 = include("engines.wine.verbs.dotnet472");
-const Vcrun2017 = include("engines.wine.verbs.vcrun2017");
+const dotnet472 = include("engines.wine.verbs.dotnet472");
+const vcrun2017 = include("engines.wine.verbs.vcrun2017");
 const DXVK = include("engines.wine.verbs.dxvk");
-const FAudio = include("engines.wine.verbs.faudio");
-
 const OverrideDLL = include("engines.wine.plugins.override_dll");
 
+
 new SteamScript()
     .name("Space Engineers")
     .editor("Keen Software House")
     .author("Zemogiter")
-    .appId("244850")
-    .wineVersion("4.14")
+    .appId(244850)
+    .wineVersion("4.15")
     .wineDistribution("upstream")
     .wineArchitecture("amd64")
-    .preInstall(function (wine) {
-        new DotNET472(wine).go();
-        new Vcrun2017(wine).go();
+    .preInstall(function (wine, wizard) {
```

Comment:
Please remove the `wizard` parameter from the function statement. Use 

```javascript
const wizard = wine.wizard();
```

instead